### PR TITLE
Setup the travis configuration

### PR DESCRIPTION
**Pull request by yamlin**

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

**Comment by chinkong83** [comment]
👍

**Comment by shichung** [comment]
LGTM

File: .travis.yml [hunk 1]
```diff
@@ -1,15 +1,18 @@
...
-    - $HOME/.gradle/wrapper/
-
+  - $HOME/.gradle/caches/
+  - $HOME/.gradle/wrapper/
 after_success:
-  - ./gradlew coveralls
\ No newline at end of file
+- ./gradlew coveralls
+script:
+- test "${TRAVIS_PULL_REQUEST}" != "false" || test "${TRAVIS_BRANCH}" != "master" || ./gradlew bintrayUpload
```
Comment (shichung) [inline .travis.yml]: "${TRAVIS_PULL_REQUEST}" != "false"

does it equal  "${TRAVIS_PULL_REQUEST}" == "true"  ?

Comment (yamlin) [inline .travis.yml]: yes, I will change to 
"${TRAVIS_PULL_REQUEST}" == "true"

Comment (shichung) [inline .travis.yml]: thanks

**Comment by shichung** [comment]
LGTM

**Comment by shichung** [comment]
https://travis-ci.org/yahoo/parsec-libraries

**Pull request merged**